BILL_OF_MATERIALS.md

```diff
@@ -0,0 +1,43 @@
+# Bill Of Materials (BOM)
+
+
+## Alexander Koch arms
+
+### Follower and Leader arms
+
+TODO
+
+### Leader arm only
+
+| Part | Amount | Cost/p US | Buy US | Cost/p EU | Buy EU | Cost/p UK | Buy UK |
```

Comment:
Nit: What do you think about having the Manufacturer and part number, e.g, Waveshare Serial Bus Servo Driver Board would have Wonrabai and B0CTMM4LWK that way it may be easier for people outside of these countries to easily search these part?